### PR TITLE
add badges to readme

### LINKED ISSUE #45: Update readme.md
- Styling readme to make it look attractive
- Adding badges and live stats
- Writing proper description

### PR DESCRIPTION
**Pull request by tejan-singh**
fixes #45

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # DarkSite
 
-## What is DarkSite?
+[![forthebadge](https://forthebadge.com/images/badges/built-with-love.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/built-by-developers.svg)](https://forthebadge.com)
```
Comment (sudo-rgorai) [inline README.md:3]: Don't you think that after `built-with-love` adding `built-by-developers` would be a bit redundant? Can we remove this?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+[![forthebadge](https://forthebadge.com/images/badges/made-with-javascript.svg)](https://forthebadge.com)
+[![forthebadge](https://forthebadge.com/images/badges/built-with-swag.svg)](https://forthebadge.com)
+
```
Comment (sudo-rgorai) [inline README.md:5]: Again, `built-with-swag` also seems a little repetitive. Do you think we should remove this?

File: README.md [hunk 3]
```diff
@@ -16,12 +26,17 @@ It is a chrome extension that customizes your new tab to a minimal dark theme.
 6. Browse and select the repository directory.  
   
   
-That's it. You're done! Welcome to the Dark Side.  
+That's it. You're done! Welcome to the Dark Side.
+
+## Tech Stack Used:
++ HTML, CSS, JavaScript
```
Comment (sudo-rgorai) [inline README.md:30]: I think `Tech Stack Used:` can be replaced with `Tech Stack`.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,16 @@
...
+[![forthebadge](https://forthebadge.com/images/badges/built-with-swag.svg)](https://forthebadge.com)
+
+[![GitHub issues](https://img.shields.io/github/issues/sudo-rgorai/DarkSite?color=Green&style=for-the-badge)](https://github.com/sudo-rgorai/DarkSite/issues)
+![GitHub closed pull requests](https://img.shields.io/github/issues-pr-closed-raw/sudo-rgorai/DarkSite?style=for-the-badge)
+
+
+
+## About Project
 
 It is a chrome extension that customizes your new tab to a minimal dark theme.
```
Comment (sudo-rgorai) [inline README.md]: If we are replacing `What is DarkSite` with `About Project` then we should replace `It is a chrome extension...` with `DarkSite is a chrome extension...`

**Review by sudo-rgorai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tejan-singh) [reply README.md:3]: I think It looks good but Its your project. If you don't like it then we can remove extra badges.

**Comment by tejan-singh** [comment]
Made Requested Changes.

**Pull request merged**